SPECS/kernel-headers/kernel-headers.spec

```diff
@@ -78,7 +78,7 @@ done
 * Tue Feb 27 2024 Chris Gunn <chrisgun@microsoft.com> - 6.6.14.1-3
 - Bump release to match kernel
 
-* Tue Jan 30 2024 Cameron Baird <cameronbaird@microsoft.com> - 6.6.14.1-2
+* Thu Feb 22 30 2024 Cameron Baird <cameronbaird@microsoft.com> - 6.6.14.1-2
```

Comment:
Feb 22 was the date this PR merged - https://github.com/microsoft/azurelinux/pull/7618